be/src/formats/csv/csv_reader.cpp

```diff
@@ -589,6 +589,7 @@ void CSVReader::split_record(const Record& record, Fields* columns) const {
             }
         }
     } else {
+        DCHECK(_column_delimiter_length > 0);
```

Comment:
change this variable to constant and check at beginning

Reply:
fixed